src/pretix/base/forms/auth.py

```diff
@@ -68,6 +71,7 @@ class RegistrationForm(forms.Form):
     error_messages = {
         'duplicate_email': _("You already registered with that e-mail address, please use the login form."),
         'pw_mismatch': _("Please enter the same password twice"),
+        'pw_invalid': _("Please enter a valid password")
```

Comment:
I think this line is unnecessary as we never access it because ``password_validators_help_texts()`` already gives a human-readable message.